packages/block-editor/src/components/background-image-control/index.js

```diff
@@ -369,15 +371,28 @@ function BackgroundImageControls( {
 							displayInPanel,
 					} ),
 				} }
-				name={
-					<InspectorImagePreviewItem
-						className="block-editor-global-styles-background-panel__image-preview"
-						imgUrl={ url }
-						filename={ title }
-						label={ imgLabel }
-					/>
-				}
 				variant="secondary"
+				renderToggle={ ( { isOpen, onToggle } ) => (
+					<Button
+						__next40pxDefaultSize
+						aria-expanded={ isOpen }
+						aria-haspopup="true"
+						onClick={ onToggle }
+						onKeyDown={ ( event ) => {
+							if ( event.keyCode === DOWN ) {
```

Comment:
I know we've been doing it like that in other places, but `event.keyCode` is [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode). You might want to use:

```suggestion
							if ( event.key === 'ArrowDown' ) {
```

Reply:
Hi @tyxla, thanks for the review. I've incorporated the suggested changes in the latest commit.